module/icp/io/sha1_mod.c

```diff
@@ -360,11 +360,11 @@ sha1_digest_final_uio(SHA1_CTX *sha1_ctx, crypto_data_t *digest,
 			 * the user only what was requested.
 			 */
 			SHA1Final(digest_scratch, sha1_ctx);
-			bcopy(digest_scratch, (uchar_t *)uio_iovbase(digest->
-			    cd_uio, vec_idx) + offset,
-			    digest_len);
+			bcopy(digest_scratch,
+			    (uchar_t *)zfs_uio_iovbase(digest->cd_uio, vec_idx)
+			    + offset, digest_len);
```

Comment:
I did this originally because the + made the line longer than 80 characters. I updated how this is split between lines. Double check it though with the new way I split this up between lines.